debian/control

```diff
@@ -7,12 +7,11 @@ Build-Depends:
  dh-golang (>= 1.1),
  golang-go (>= 1.7),
  git,
- golang-github-aws-aws-sdk-go-dev (>= 1.16.26),
+ golang-github-aws-aws-sdk-go-dev (>= 1.36.0),
```

Comment:
I wonder if there's tooling to align the required packages with the `go.mod` specified versions (which I'm guessing is why this was force-pushed to [match 1.36.0](https://github.com/awslabs/amazon-ecr-credential-helper/pull/264/commits/390d97528616b4b5a507b1d871c1d49380fb4edc) instead of 1.35.4). This would have been an easy oversight and could have slipped through.

Maybe @nmeyerhans would know if such tooling exists?

Reply:
> which I'm guessing is why this was force-pushed to match 1.36.0 instead of 1.35.4

:sweat_smile: yes...I had most of this top commit written prior to determining whether the release would include https://github.com/awslabs/amazon-ecr-credential-helper/pull/253, which is why I had the mismatch...

Reply:
Note that unlike with Go modules, the dependencies here follow Debian's/Apt's style of version matching.  That means that even though the dependency was specified as `>= 1.35.4`, it was actually being built against [1.36.33-1](https://packages.debian.org/source/unstable/golang-github-aws-aws-sdk-go) (the latest present in `unstable`, since that's where I do my test builds).